bundles/nl.esi.comma.abstracttestspecification.tests/resources/input/issue249/issue249.atspec

```diff
@@ -0,0 +1,135 @@
+import "issue249.ps"
+
+using issue249.ContextIssueProducer.part1
+using issue249.ContextIssueProducer.part2
+using issue249.ContextIssueProducer.report
+using issue249.ContextIssueProducer.message
+using issue249.ContextIssueProducer.configuration
+using issue249.ContextIssueProducer.Event_1p0urs4
+using issue249.ContextIssueProducer.Event_0m7n0r6
+using issue249.ContextIssueProducer.between_Start_and_ComposeContext
+using issue249.ContextIssueProducer.between_ComposeContext_and_RunContext
+using issue249.ContextIssueProducer.between_RunContext_and_ComposeLocal
+using issue249.ContextIssueProducer.between_ComposeLocal_and_RunLocal
+using issue249.ContextIssueProducer.between_RunLocal_and_Compose
+using issue249.ContextIssueConsumer.Event_0fmv4m5
+
+abstract-test-definition
+
+Test-Scenario: S1
+
+compose-step-name: ContextIssueProducer_ComposeContext_default_0
+action-case: issue249.ContextIssueProducer.ComposeContext.default
+input-binding:
+between_Start_and_ComposeContext : {
+    "id": 0,
+    "prod_field": "Hello Consumer!"
+}
+output-data:
+between_ComposeContext_and_RunContext : {
+    "id": 0,
+    "prod_field": "Hello Consumer!"
+}
+suppress(between_ComposeContext_and_RunContext.id)
+
+run-step-name: ContextIssueProducer_RunContext_default_0 step-type: "ProducerRunType"
+action-case: issue249.ContextIssueProducer.RunContext.default
+consumes-from-step: ContextIssueProducer_ComposeContext_default_0 { between_ComposeContext_and_RunContext }
```

Comment:
@dbera I manually added this line to the atspec, please review if this is correct. This lineto be generated by the Python code.